### ISSUE TITLE
ASB 2019.08

### ISSUE THREAD

**Issue by MSe1969** [cm12-amami/discussion#56]
And another month...

**Comment by MSe1969** [comment]
***2019-08-01 ASB***
**Android Runtime**
- [x] CVE-2019-2120   A-130821293   EoP   High - applied

**Framework**
- n/a CVE-2019-2121  A-131105245   EoP   High - Android 9 only
- [x] CVE-2019-2122  A-127605586 [2][3]  EoP  High - applied
- n/a CVE-2019-2125  A-132275252  EoP  Moderate - no ChangeDefaultDialerDialog.java

**Media Framework**
- [x] CVE-2019-2126  A-127702368 [2]  RCE  High - applied
- [x] CVE-2019-2128  A-132647222  EoP  High - applied
- [x] CVE-2019-2129  A-124781927  ID  High - applied

**System**
- [x] CVE-2019-2130  A-132073833  RCE  Critical - applied
- n/a CVE-2019-2131  A-119115683  EoP  High - code differs significantly
- [x] CVE-2019-2132  A-130568701  EoP  High - applied
- n/a CVE-2019-2133  A-132082342  EoP  High - no phNxpExtns_MifareStd.c
- n/a CVE-2019-2134  A-132083376  EoP  High - no phNxpExtns_MifareStd.c
- n/a CVE-2019-2135  A-125900276  ID  High - no phNxpExtns_MifareStd.c
- [x] CVE-2019-2136  A-132650049 [2] [3] [4]  ID  High - applied
- n/a CVE-2019-2137  A-132438333 [2]  DoS  High - Android 9 only

***2019-08-05 ASB***
**Media Framework**
- n/a CVE-2019-2127  A-124899895  EoP  High - files not present

**Qualcomm**
- n/a CVE-2019-10492  A-132170519 QC-CR#2389432  N/A  Critical - bootloader
- n/a CVE-2019-10509  A-132171185 QC-CR#2359039  N/A  High - coding differs
- n/a CVE-2019-10510  A-132173563 QC-CR#2305025  N/A  High - code not present
- n/a CVE-2019-10499  A-134440231 QC-CR#2398099  N/A  High - no qcom_glink_smem.c
- n/a CVE-2019-10538  A-132193791 QC-CR#2448763  N/A  High - no icnss.c

**Comment by MSe1969** [comment]
Test build running...

**Comment by MSe1969** [comment]
Test build looks good - want to test as well or should I ship, once my test is finished?

**Comment by Micha-Btz** [comment]
Ship it, I'm not ready to test.

**Comment by MSe1969** [comment]
Shipped, test branch merged into stable, so closing...
